src/tensor.cpp

```diff
@@ -52,6 +52,7 @@ bool IsDataTypeSupported(miopenDataType_t t)
     case miopenInt8:
     case miopenBFloat16:
     case miopenDouble: return true;
+    case miopenInt64: return false;
```

Comment:
Why is this set to false?

Reply:
Because adding support for tensor<T> class was out of scope for this PR. But probably have it as true also possible